Data/EnumMap/Base.hs

```diff
@@ -198,6 +198,12 @@ instance (Enum k, Read k, Read a) => Read (EnumMap k a) where
     list <- readPrec
     return (fromList list)
 
+instance (ToJSON a) => ToJSON (EnumMap k a) where
+    toJSON = toJSON . unWrap
+    toEncoding = toEncoding . unWrap
+
+instance (FromJSON a) => FromJSON (EnumMap k a) where
+    parseJSON = fmap (EnumMap . I.fromList) . parseJSON
```

Comment:
`FromJSON1` and `FromJSON2` instances may be useful too.

Reply:
I ll check it